code/modules/surgery/surgery_step.dm

```diff
@@ -202,9 +202,13 @@
  */
 //SKYRAT EDIT START: Fixes painkillers not actually stopping pain.
 /datum/surgery_step/proc/display_pain(mob/living/target, pain_message, mechanical_surgery = FALSE)
-	if(HAS_TRAIT(target, TRAIT_NUMBED) || target.stat >= UNCONSCIOUS)
+	if(target.stat >= UNCONSCIOUS || mechanical_surgery == TRUE) //robots and the unconscious do not worry about pain
```

Comment:
not keen on more stupid synthetic buffs

Reply:
Hopefully addressed with latest commit - robots really have no reason to not just Sleep even if they can't process numbing reagents, so they now get a penalty that falls in between the other two if they don't.